lib/hariko/resource/parser.js

```diff
@@ -98,6 +96,15 @@ exports.parse = function (rawdata) {
               entry.response.statusCode);
           }
         }
+
+        if (initialEntry != null) {
+          var entry = initialEntry
```

Comment:
__[jshint]__
* 'entry' is already defined.
* Missing semicolon.

<!-- Comment by [SideCI](https://sideci.com) -->